### DISCUSSION TITLE
FCM Key is not set. Please tell me how to fill it.

### DISCUSSION THREAD

**Discussion by ITITIT123** [LiveHelperChat/livehelperchat#1884] [Q&A]
xx3FCM Key is not set. Xx3fcm key is not set

**Comment by remdex** [comment]
And how it has to do anything with Live Helper Chat

**Reply by ITITIT123** [reply]
Google firebase cloud message Do I need to apply on my own? Or do you use the system default?

**Reply by remdex** [reply]
You need to generate your own or follow and comment out temporary FCM related parts.
https://doc.livehelperchat.com/docs/mobile#how-to-compile

**Reply by ITITIT123** [reply]
I need to use your Google firebase cloud message is that OK, redevelop, I don't have the technology. I need to use yours

**Reply by ITITIT123** [reply]
Do not know how to compile, do not know how to compile, can teach how to modify the file

**Reply by remdex** [reply]
That's for you to learn all that, not me to teach... Refer to Flutter documentation
